igvc_navigation/src/path_follower/main.cpp

```diff
@@ -22,17 +22,40 @@ SmoothControl controller;
 
 void path_callback(const nav_msgs::PathConstPtr& msg)
 {
-  ROS_INFO("Follower got path");
+  ROS_INFO_STREAM("Follower got path. Size: " << msg->poses.size());
   path = msg;
 }
 
 double get_distance(double x1, double y1, double x2, double y2)
 {
+  /**
+  Calculates euclidian distance between two points
+  */
   return sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2));
 }
 
+void compute_angle(float& angle, Eigen::Vector3d vec2, Eigen::Vector3d vec1)
```

Comment:
This one I think we could also extract out of here and into robot_state to make this class cleaner

Reply:
This isn't a class. 

This helper method could definitely be placed in a separate class, although i'm not sure whether it should be put in `RobotState` or `NodeUtils`.

Reply:
as is node utils, then make a RobotState method that gives it to you for the states

Reply:
I'm not too sure what you mean by "as in node utils", could you explain what you mean?

Reply:
He means the way it is now it should be put in NodeUtils. I agree since it is a general operation that could be used by a lot of nodes, not just the ones relating to state

Reply:
as is, put it in node utils.